### DISCUSSION TITLE
How can I have margin notes?

### DISCUSSION THREAD

**Discussion by vrilcode** [sile-typesetter/sile#1809] [Q&A]
Hi, am new to SILE and quite impressed. What I need is margin note functionality. I've seen the sidenotes example in the manual, but I don't know how to use it in my SILE document. It would be perfect, if I could "reformat" subsubsections, so that they are printed as margin notes. Is that possible? Any help appreciated.

**Comment by Omikhleia** [comment]
As @vrilcode I'd like to see a working margin note functionality. The example code in the manual seems wrong for any generic solution...
